python/lsst/ci/build.py

```diff
@@ -111,6 +125,8 @@ def __init__(self, build_dir, manifest, progress, eups):
         self.manifest = manifest
         self.progress = progress
         self.eups = eups
+        self.built = []
+        self.failed_at = False
```

Comment:
I would find it less confusing if this was set to `None`. Later down in `write_status` the `is not False` construct takes some thought to parse until you realize that `failed_at` is not a boolean. `None` seems like a reasonable default to indicate that the build did not fail.

Reply:
Agreed -- I'm not sure why I did it that.  Converting it to `None`.